specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/OperationsManagement.json

```diff
@@ -69,8 +69,8 @@
                     }
                 ],
                 "responses": {
-                    "200": {
-                        "description": "OK response definition.",
+                    "201": {
```

Comment:
Please verify if this is a long running operation

Reply:
Yes, this is a long running operation.

Reply:
Please mark it as long running (`"x-ms-long-running-operation":true`)
While we are at it please confirm whether the `DELETE` operation is long running too and update it